clients/nexus-iq/src/main/kotlin/NexusIqService.kt

```diff
@@ -95,7 +95,7 @@ interface NexusIqService {
     data class SecurityIssue(
         val reference: String,
         val severity: Float,
-        val url: URL
+        val url: URL?
```

Comment:
I wonder if it would anyway make sense to turn it into a `String`.
Then the client code can decide whether it wants to use `URL` or `Uri` or whatever it wants to work with it.
Then the `null` case could be turned into `empty string` which might be easier to handle.
Any thoughts on this?

Reply:
I'm not a fan of hiding nullability behind an empty String. Using `URL?` will force the caller to handle the possibility of a non-existent URL. With an empty string it might just lead to an unhandled error.
Therefore I'd like to keep it as-is.